src/main/scala/uk/gov/nationalarchives/notifications/messages/Messages.scala

```diff
@@ -127,8 +127,9 @@ object Messages {
     incomingEvent match {
       case ev: ExportStatusEvent if ev.environment == "prod" && ev.successDetails.exists(_.consignmentType == "judgment") =>
         Seq(eventConfig("slack.webhook.judgment_url"))
-      case ev: ExportStatusEvent if ev.environment == "prod" && ev.successDetails.exists(_.consignmentType == "standard") =>
-        Seq(eventConfig("slack.webhook.standard_url"))
+      case ev: ExportStatusEvent if ev.environment == "prod" &&
+        (ev.successDetails.exists(_.consignmentType == "standard") || ev.successDetails.exists(_.consignmentType == "historicalTribunal")) =>
```

Comment:
I have a small suggestion if you like it. Combine the checks in a single `exists` method:

```
case ev: ExportStatusEvent if ev.environment == "prod" &&
        ev.successDetails.exists(details => details.consignmentType == "standard" || details.consignmentType == "historicalTribunal") =>
```